.github/workflows/main.yml

```diff
@@ -32,12 +32,12 @@ jobs:
       with:
         fetch-depth: 0 # all
 
-    - name: Setup NET6.0
+    - name: Setup NET 8.0
       uses: actions/setup-dotnet@v3
       with:
-        dotnet-version: '6.0.x'
+        dotnet-version: '8.0.x'
 
-    - name: Test NET6
+    - name: Test NET 6
```

Comment:
If the tests are targeting both .net6 and net8, then you should have "Setup" for each of them before proceeding.

You get away with it because net6 is preinstalled on the standard github build image, but that won't be the case forever, it's good to add explicit setup for the things we use